app/templates/views/confirm_email_address.html

```diff
@@ -0,0 +1,42 @@
+{% extends "document_download_template.html" %}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{%- from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary -%}
+
+
+{% block per_page_title %}
+  Confirm your email address
```

Comment:
We should have some conditional logic in here to add an 'Error: ' prefix to the page title if there are errors in the page, to follow design system guidance: https://design-system.service.gov.uk/patterns/validation/#how-to-tell-the-user-about-validation-errors

Looks like that might mean either not using `per_page_title` or overriding it here with `pageTitle` from the base layout (see https://design-system.service.gov.uk/styles/page-template/#options).

Reply:
I've added the `Error: ` prefix by adding some conditional logic to the content passed to the `per_page_title` block. Since `per_page_title` is only being used for the title and not also for the main heading (unlike other pages) I could still use it. (I'm providing the content for the H1 separately by using the `content` block and not the `main_content` block in order to put the heading below the error summary.)